assets/arc-0052/package.json

```diff
@@ -15,11 +15,13 @@
     "@types/libsodium-wrappers-sumo": "^0.7.7",
     "@types/node": "^20.7.1",
     "ts-jest": "^29.1.1",
+    "tweetnacl": "^1.0.3",
     "typescript": "^5.2.2"
   },
   "dependencies": {
     "ajv": "^8.12.0",
     "algo-msgpack-with-bigint": "^2.1.1",
+    "algosdk": "^2.7.0",
```

Comment:
Can we not import `algosdk` it has too many third-party dependencies on it's own.

Reply:
so make `signAlgoTransaction` accept an encoded transaction?
we'll need to change the Kotlin impl to match this behaviour as well